src/Features/CSharp/Portable/Completion/CSharpCompletionService.cs

```diff
@@ -22,7 +22,6 @@ internal class CSharpCompletionService : AbstractCompletionService
             {
                 new AttributeNamedParameterCompletionProvider(),
                 new NamedParameterCompletionProvider(),
-                new SpeculativeTCompletionProvider(),
```

Comment:
I think the provider itself does not depend on the order. But the `AbstractCompletionService` does a `MergeAndPruneCompletionLists` method which has some interesting logic generally prefers the later item.